gradle/base-information.gradle

```diff
@@ -32,8 +32,9 @@ ext {
 		ormVersion = HibernateVersion.fromFile(ormVersionFile, project)
 	}
 
+	def version = System.getProperty( 'jakartaJpaVersion', '3.0.0' )
```

Comment:
Should this be `3.1`?  Note that there is no Persistence `3.0.0` release, that would be `3.0`.

Reply:
https://search.maven.org/artifact/jakarta.persistence/jakarta.persistence-api/3.0.0/jar

Reply:
I'm wrong, the SPEC version is `3.0` but the maven artifact is definitely `3.0.0` as also shown on https://jakarta.ee/specifications/persistence/3.0:
```
Maven coordinates

    [jakarta.persistence:jakarta.persistence-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.persistence/jakarta.persistence-api/3.0.0/jar)
```